src/VirtualClient/VirtualClient.Dependencies/MsMPIInstallation.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace VirtualClient.Dependencies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO.Abstractions;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.DependencyInjection;
+    using VirtualClient.Common;
+    using VirtualClient.Common.Extensions;
+    using VirtualClient.Common.Telemetry;
+    using VirtualClient.Contracts;
+
+    /// <summary>
+    /// Installation component for the MS-MPI.
+    /// </summary>
+    public class MSMPIInstallation : VirtualClientComponent
+    {
+        private const string LockFileName = "msmpisuccess.lock";
+
+        private ISystemManagement systemManagement;
+        private IPackageManager packageManager;
+        private ProcessManager processManager;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MSMPIInstallation"/> class.
+        /// </summary>
+        /// <param name="dependencies">An enumeration of dependencies that can be used for dependency injection.</param>
+        /// <param name="parameters">A series of key value pairs that dictate runtime execution.</param>
+        public MSMPIInstallation(IServiceCollection dependencies, IDictionary<string, IConvertible> parameters)
+            : base(dependencies, parameters)
+        {
+            this.systemManagement = dependencies.GetService<ISystemManagement>();
+            this.packageManager = dependencies.GetService<IPackageManager>();
+            this.processManager = this.systemManagement.ProcessManager;
+        }
+
+        /// <summary>
+        /// Installs the MS-MPI.
+        /// </summary>
+        protected async override Task ExecuteAsync(EventContext telemetryContext, CancellationToken cancellationToken)
+        {
+            if (this.Platform == PlatformID.Win32NT)
```

Comment:
Implement the IsSupported method. If we only support Windows, the installation should no-op